src/components/paintings-page/collections-drop-down/collections-drop-down.jsx

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useRef, useState } from 'react';
+import styles from './collections-drop-down.module.scss';
+import DropDown from '@/components/shared/form-components/drop-down/drop-down';
+
+export default function CollectionsDropDown() {
+  const listItems = [
+    'All',
+    'Landscapes',
+    'Abstracts',
+    'Floral',
+    'Portal',
+    'Smalls',
+  ];
+  const [isToggled, setIsToggled] = useState(false);
+  const [dropDownValue, setDropDownValue] = useState('');
+  const inputRef = useRef(null);
+
+  useEffect(() => {
+    const updateInputValue = () => {
+      inputRef.current.value = dropDownValue;
+      setIsToggled(false);
+    };
+    if (dropDownValue) {
+      updateInputValue();
+    }
+  }, [dropDownValue]);
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.borderTop}>
+        <p className={styles.filterBy}>Filter By</p>
+        <DropDown
+          isForm={false}
```

Comment:
do you mind changing the name of this property to `isReactHookForm`? That helps me understnad wht it is way more.